Handle arguments to anonymous constructors in `convertCallArguments` (#416)
eisop/checker-framework: docs/CHANGELOG.md

**User-visible changes:**

> The format of the arguments of an `ObjectCreationNode` is correct when the node refers to an
> anonymous constructor with an explicit enclosing expression.

Reviewer: ```suggestion
Corrected the arguments to an `ObjectCreationNode` when the node refers to an
anonymous constructor invocation with an explicit enclosing expression.
```

Reviewer: This should move down to `Implementation details`.

Author: Thanks! Done in my local.

**Implementation details:**

Corrected the arguments to an `ObjectCreationNode` when the node refers to an

> anonymous constructor invocation with an explicit enclosing expression in Java11+.

Reviewer: ```suggestion
anonymous constructor invocation with an explicit enclosing expression in Java 11+.
```
